cli/cli_test.go

```diff
@@ -64,7 +67,7 @@ func (c cliTest) Run(line string) {
 	args = append(args, a[1:]...)
 
 	fmt.Fprintf(os.Stderr, "%s\n", args)
-	fmt.Println(line)
+	fmt.Println(strings.Join(a, " "))
```

Comment:
can you give this a better than name "a"? also, i think this should be `Fprintf(osStderr...`, but I could be missing the reason it isnt that already.